packages/component-library/src/components/layout/mt-inset/mt-inset.vue

```diff
@@ -0,0 +1,12 @@
+<template>
```

Comment:
Would it make more sense to set the `inset` mode via a prop?

Reply:
Yes, it does. I will do that.